test/e2e/cases/basic.go

```diff
@@ -101,6 +107,15 @@ func testHTTPSIngress(project, kubeConfigPath, lbName string, kubectlArgs []stri
 	fmt.Println("PASS: got 200 from ingress url")
 	testList(project, ipAddress, lbName)
 
+	// Running create again should not return any error.
+	/*
+		TODO(nikhiljindal): Uncomment this once https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/125 is fixed.
+		_, err := createIngress(project, kubeConfigPath, lbName, "examples/zone-printer/ingress/https-ingress.yaml")
+		if err != nil {
+			glog.Fatalf("Unexpected error in re-creating ingress: %+v", err)
```

Comment:
How about running the re-create, but just log ERROR until the bug is fixed.

Reply:
Good idea, done.
Changed to Info log as expected error.